wpilibNewCommands/src/main/native/include/frc2/command/button/Trigger.h

```diff
@@ -339,8 +339,8 @@ class Trigger {
    * @deprecated Use WhileTrue(Command) and construct a RunCommand manually
    */
   WPI_DEPRECATED("Use WhileTrue(Command) and construct a RunCommand manually")
-  Trigger WhileActiveContinous(std::function<void()> toRun,
-                               std::span<Subsystem* const> requirements = {});
+  Trigger WhileActiveContinuous(std::function<void()> toRun,
```

Comment:
This changes an already-deprecated method name.  Let’s just leave it as is for now, since we are likely to remove the function entirely in the future.